examples/fishnet_api/requests.py

```diff
@@ -47,6 +47,11 @@ class RequestExecutionResponse(BaseModel):
     unavailableTimeseries: Optional[List[Timeseries]]
 
 
-class DenyPermissionsResponse(BaseModel):
+class FishnetResponseDataset(BaseModel):
```

Comment:
replace occurrences of this class with custom classes for every returned response type